cmd/get/util.go

```diff
@@ -0,0 +1,75 @@
+package get
+
+import (
+	"github.com/flyteorg/flytectl/cmd/config"
+	"github.com/flyteorg/flytectl/pkg/filters"
+	"github.com/flyteorg/flyteidl/gen/pb-go/flyteidl/admin"
+)
+
+func buildResourceListRequestWithName(c *config.Config, name string) *admin.ResourceListRequest {
+	fieldSelector, err := filters.Transform(filters.SplitTerms(c.FieldSelector))
+	if err != nil {
+		fieldSelector = ""
+	}
+	request := &admin.ResourceListRequest{
+		Limit:   uint32(c.Limit),
+		Filters: fieldSelector,
+		Id: &admin.NamedEntityIdentifier{
+			Project: c.Project,
+			Domain:  c.Domain,
+			Name:    name,
+		},
+	}
+	if sort := buildSortingRequest(c); sort != nil {
+		request.SortBy = sort
+	}
+	return request
+}
+
+func buildResourceListRequestWithoutName(c *config.Config) *admin.ResourceListRequest {
```

Comment:
nit: since it seems like the only difference between the above and this is the missing name in the NamedEntityIdentifier could we maybe refactor to have both call a shared method that accepts a NamedEntityIdentifier and centralizes how we build the ResourceListRequest in one place?